docs/rst/amldashboard/user_manual/interfaces.rst

```diff
@@ -0,0 +1,141 @@
+   It features two video representations:
+
+   - **Ground Truth Video Stream**: Shows the actual video feed.
+
+   - **Overlaid Skeleton Video**: Displays the skeleton recognized by the hand recognition algorithm.
+
+   These visual aids allow users to test and explore the aspects captured by the model.
+
+2. **Collecting Data**: Users can create their own data sets to train an |aml| model.
+   A drop-down menu allows the selection of one out of four musical instruments for demonstration purposes.
+
+3. **Data Set Overview**: Each collected image is displayed in a data set overview, where users can explore, move, or delete images in the training stack.
+   This feature is crucial for iterative training, enabling users to remove or add new labeled data to improve the |aml| algorithm as needed.
+
+Training
+========
+
+The training page is separate from the data collection page to educate users on the various stages of the :term:`ML` life-cycle.
+
+It is divided into two sections:
+
+1. **AML Model Training Information (Left Side)**:
+
+   - **Training Information**: Users can set parameters such as the number of iterations.
+
+   - **Training Visualization**: Displays the current state of training.
+
+2. **Comparison Model Training (Right Side)**:
+
+   - **Model Parameters**: Users can set or alter parameters for a comparison model, such as a Multi-layer Perceptron (MLP) classifier.
+     This model can be replaced with any other type of classifier for direct comparison with |aml|.
+
+   - **Training Progress**: A loading bar shows the projected training duration.
+     Below it, variations in loss and accuracy per epoch are visualized, helping experts assess the classification quality and data noise level.
+
+Fetching
+========
+
+Retrieve trained |aml| statistics and Models effortlessly.
+It includes options to search for |aml| statistics and request an |aml| model.
+
+It is divided into the following sections:
+
+1. **Status Display**: Shows the current status of collaborative learning, statistics, and model fetching.
+
+2. **AML Statistics Fetcher**:
+
+   - **Search Button**: Search for |aml| statistics.
+
+   - **Statistics Display**: Display the fetched statistics
+
+3. **AML Model Fetcher**:
+
+   - **Request Button**: Request an |aml| model.
+
+   - **Model Display**: Display the fetched model.
+
+Explore Models
+==============
+
+The model exploration interface provides space for developers to add any desired model comparison matrix.
+The Marcelle framework supports various comparison modules and allows users to create custom :term:`GUI` elements for specific calculations.
+
+In the current setup, a confusion matrix is provided based on the trained model.
+Users must actively trigger the update of the matrix, facilitating a clear before-and-after comparison.
+Future work aims to extend this section with more targeted visualizations utilizing the |aml| output structure.
+
+Music Exploration
```

Comment:
I will change the name to a more generic term since it’s meant to describe the functionality of the interface rather than focusing on specific contexts like music.

Reply:
Done